src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends CrudRepository<Book, Long> {
+
+  Book findFirstByAuthor(String author);
```

Comment:
A good practice is to return `Optional<Book>` instead of the Book by itself, which will allow you to prevent null checks in the future 😄 
If you need more info on this, check this out: https://www.baeldung.com/java-optional :)